android/pytorch_android/src/main/java/org/pytorch/Module.java

```diff
@@ -4,22 +4,45 @@
 
 import com.facebook.jni.HybridData;
 
+/**
+ * Java holder for torch::jit::script::Module which owns it on jni side.
+ */
 public class Module {
 
   private NativePeer mNativePeer;
 
+  /**
+   * Loads serialized to file torchscript module from specified absolute path on the disk.
```

Comment:
"Loads serialized torchscript module from the specified absolute path on the disk."